buildstockbatch/schemas/v0.3.yaml

```diff
@@ -19,14 +19,14 @@ postprocessing: include('postprocessing-spec', required=False)
 references: map(required=False)
 ---
 gcp-spec:
-  job_identifier: regex('^[a-zA-Z_-]\w{,20}$', required=True)
-  gcp_project: str(required=True)
+  job_identifier: regex('^[a-zA-Z_]\w{,9}$', required=True)
+  project: str(required=True)
   region: str(required=True)
   artifact_registry: include('gcp-ar-spec', required=True)
   batch_array_size: num(min=1, max=10000, required=True)
   notifications_email: regex('^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$', name='email', required=True)
   gcs: include('gcs-spec', required=True)
-  use_spot: bool(required=False)
+  use_spot: bool(required=True)
```

Comment:
Why is it required (and inconsistent with aws-spec)?

Reply:
The aws-spec is wrong. 🙂  The job fails ([here](https://github.com/NREL/buildstockbatch/blob/661f66abc1e78faf47c9ba56c07ba0918278a15c/buildstockbatch/aws/aws.py#L1696)) if it doesn't exist.

Reply:
(As long as we're diverging) I think it'd be better to leave the config optional and make the code work with it non-present (looks like you can use [`get(key[, default])`](https://docs.python.org/3/library/stdtypes.html#dict.get)).